dts/bindings/mmc/nxp,imx-usdhc-slot.yaml

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2019, NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: NXP i.MX USDHC slot
+version: 0.1
+
+description: >
+    NXP i.MX USDHC slot
```

Comment:
I would propose to use a generic mmc.yaml which will contains the properties like compatible, address_cells, size_cells, label, which will be common for different mmc/sdhc controllers. nxp,imx-usdhc.yaml should inherit mmc.yaml and contain properties like reg, interrupts, pwr-gpios, cd-gpios and compatible's constrain.

Reply:
I agree to use generic mmc.yaml which nxp,imx-usdhc.yaml inherits.
But pwr-gpios are cd-gpios are not part of usdhc IP but part of board design.
So we  may have mmc.yaml->nxp,imx-usdhc.yaml(SoC)->nxp,imx-usdhc-slot.yaml(board).
Is it OK?

Reply:
is there anythong particular to nxp,imx-usdhc.yaml ? if not, you can skip and just do mmc.yaml->nxp,imx-usdhc-slot.yaml(board)

Reply:
OK, I ill skip nxp,imx-usdhc.yaml

Reply:
Better to keep as close to linux dts files as possible. We could add cd-gpios and pwr-gpios as optional properties in mmc.yaml. Something similar is done in linux mmc dts binding regarding cd-gpios. Below is a part of linux mmc dts binding documentation:
```
Card detection:
If no property below is supplied, host native card detect is used.
Only one of the properties in this section should be supplied:
  - broken-cd: There is no card detection available; polling must be used.
  - cd-gpios: Specify GPIOs for card detection, see gpio binding
  - non-removable: non-removable slot (like eMMC); assume always present.
```

Reply:
OK, I will use generic mmc.yaml to specify sdhc/mmc and add gpios as optional.

Reply:
Done